airbyte-integrations/connectors/source-amplitude/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_amplitude"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "==0.52.0"
+airbyte-cdk = "^0"
```

Comment:
It's way better to have `>=0.52.0`, but not blocking the update overall.

Reply:
Thanks for the review! I think we're specifically using the `^0` notation to prevent automatic major version updates, but I'll double check with Natik to confirm before unpinning the rest of the connectors 👍